services/keycloak/startup-scripts/00-configure-lagoon.sh

```diff
@@ -124,6 +124,25 @@ function configure_realm_settings {
 
 }
 
+function configure_lagoon_redirect_uris {
+  # this will always run, and will always ensure that the redirect uris are up to date
+  # changes to redirect uris should be made via the chart/envvars
+  # the value of this variable is a comma separated list of redirect uris
+  # eg LAGOON_UI_REDIRECT_URIS="http://localhost:8888/redirect1,http://localhost:8888/redirect2"
+  #
+  if [ "$LAGOON_UI_REDIRECT_URIS" != "" ]; then
```

Comment:
If this were to match our [informal pattern](https://github.com/uselagoon/lagoon/blob/main/services/keycloak/startup-scripts/00-configure-lagoon.sh#L26) it would be `KEYCLOAK_LAGOON_UI_CLIENT_REDIRECT_URIS`